ui/ui-expenses/src/screens/list/list.reactions.js

```diff
@@ -56,6 +56,7 @@ export const submit = ({ payload }, store, { window, http }) => {
           const reader2 = new window.FileReader()
           reader2.onerror = (readerE) => { store.dispatch({ type: '@@file/ON_ERROR', payload: readerE }) }
           reader2.onload = async (readerE) => {
+            store.devices.cameras.capture.set(readerE.target.result)
```

Comment:
what's the purpose ?

Reply:
to store capture for viewing on next screen.